CONTRIBUTING.md

```diff
@@ -24,13 +24,23 @@ Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
 Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
 ```
 
-To install Node (with npm) and Java locally on macOS, you can use [brew](https://brew.sh/):
+To install Node (with npm) and Java locally on macOS, we recommend [brew](https://brew.sh) with 
+[nvm](https://github.com/nvm-sh/nvm):
 
 ```bash
-brew install node
+brew install nvm
+nvm install $(cat .nvmrc)
```

Comment:
Thanks for adding this

Mind if we leave off the config file? It's the default path

```suggestion
nvm install
```

Reply:
Of course. Please excuse my ignorance!